pkg/build/nodeimage/internal/kube/source.go

```diff
@@ -38,7 +38,7 @@ func FindSource() (root string, err error) {
 	if err == nil && maybeKubeDir(pkg.Dir) {
 		return pkg.Dir, nil
 	}
-	return "", errors.New("could not find kubernetes source")
+	return "", fmt.Errorf("could not find %s module source under GOPATH=%s", ImportPath, build.Default.GOPATH)
```

Comment:
I think that is better to surface the original error, it contains more meaningful messages

Reply:
+1